Would it be possible to add a positive commission?

I'm trying to backtest a limit orders based strategy on a crypto exchange named bybit. The problem is that you get rebates when you use limit orders. But I see I can't enter positive commissions (-0.00025). Would it be possible to change this?

```
AssertionError: commission should be between 0-10%, is -0.00025
```

--- a/backtesting/backtesting.py
+++ b/backtesting/backtesting.py
@@ -664,8 +664,9 @@ class _Broker:
     def __init__(self, *, data, cash, commission, margin,
                  trade_on_close, hedging, exclusive_orders, index):
         assert 0 < cash, f"cash should be >0, is {cash}"
-        assert -.1 <= commission < .1, (f"commission should be between -10% "
-            "(e.g. market-maker's rebates) and 10% (fees), is {commission}")
+        assert -.1 <= commission < .1, \
+            ("commission should be between -10% "
+             f"(e.g. market-maker's rebates) and 10% (fees), is {commission}")
         assert 0 < margin <= 1, f"margin should be between 0 and 1, is {margin}"
         self._data: _Data = data
         self._cash = cash

--- a/backtesting/backtesting.py
+++ b/backtesting/backtesting.py
@@ -664,7 +664,7 @@ class _Broker:
     def __init__(self, *, data, cash, commission, margin,
                  trade_on_close, hedging, exclusive_orders, index):
         assert 0 < cash, f"cash should be >0, is {cash}"
-        assert 0 <= commission < .1, f"commission should be between 0-10%, is {commission}"
+        assert -.1 <= commission < .1, f"commission should be between -10% (e.g. market-maker's rebates) and 10% (fees), is {commission}"
         assert 0 < margin <= 1, f"margin should be between 0 and 1, is {margin}"
         self._data: _Data = data
         self._cash = cash

--- a/backtesting/backtesting.py
+++ b/backtesting/backtesting.py
@@ -664,7 +664,8 @@ class _Broker:
     def __init__(self, *, data, cash, commission, margin,
                  trade_on_close, hedging, exclusive_orders, index):
         assert 0 < cash, f"cash should be >0, is {cash}"
-        assert -.1 <= commission < .1, f"commission should be between -10% (e.g. market-maker's rebates) and 10% (fees), is {commission}"
+        assert -.1 <= commission < .1, (f"commission should be between -10% "
+            "(e.g. market-maker's rebates) and 10% (fees), is {commission}")
         assert 0 < margin <= 1, f"margin should be between 0 and 1, is {margin}"
         self._data: _Data = data
         self._cash = cash

